kubesae/info.py

```diff
@@ -1,24 +1,45 @@
 import invoke
 import yaml
 
-@invoke.task(default=True)
-def get_ansible_vars(c, var=None):
-    """Inspect ansible variables
+@invoke.task("print_ansible_vars")
+def print_ansible_vars(c, var=None, yaml=None, pty=True, hide=False):
+    """A command to inspect any ansible varible by environment. If no variable is specified then it will
+    print out the current k8s environment variables.
 
-    Displays Ansible variables in the specified environment. If no variable is specified
-    then it will print out the current k8s environment variables.
+    Args:
+        c (invoke.Context): [description]
+        var (string, optional): The ansible variable you want to expose. Defaults to None.
+        yaml (string, optional): An ansible path. Defaults to None.
+        pty (bool, optional): [description]. Defaults to True.
```

Comment:
These still have the `[description]` placeholder. Specifically I'm wondering how I'd know when/why to set `pty` and `hide` since I haven't used those before.

Reply:
Let me know if that helps.